src/main/java/de/tum/in/www1/artemis/service/messaging/MainInstanceMessageSendService.java

```diff
@@ -22,98 +23,87 @@ public MainInstanceMessageSendService(InstanceMessageReceiveService instanceMess
 
     @Override
     public void sendProgrammingExerciseSchedule(Long exerciseId) {
-        // No need to go through the broker, pass it directly
```

Comment:
interesting that those comment happened 🤔

Reply:
Jeah, probably through copy & paste.